plugins/source/hubspot/go.mod

```diff
@@ -0,0 +1,37 @@
+module github.com/cloudquery/cloudquery/plugins/source/hubspot
+
+go 1.19
+
+require (
+	github.com/clarkmcc/go-hubspot v0.0.0-20221010213350-20c2f9cbf936
+	github.com/cloudquery/plugin-sdk v1.25.0
```

Comment:
Can we update to the latest SDK?
See https://github.com/cloudquery/plugin-sdk/releases/tag/v1.28.0